axum-macros/src/debug_handler.rs

```diff
@@ -231,11 +231,11 @@ fn check_inputs_impls_from_request(
             };
 
             let from_request_bound = if must_impl_from_request_parts {
-                quote! {
+                quote_spanned! { span =>
```

Comment:
In syn's docs, this is always written without any extra spaces, even with the docs pointing out that it should look like that to stand out. I think we should follow that.

```suggestion
                quote_spanned! {span=>
```